druid/src/win_handler.rs

```diff
@@ -78,7 +78,7 @@ struct Inner<T> {
     app: Application,
     delegate: Option<Box<dyn AppDelegate<T>>>,
     command_queue: CommandQueue,
-    file_dialogs: HashMap<FileDialogToken, WindowId>,
+    file_dialogs: HashMap<FileDialogToken, (WindowId, Selector<FileInfo>, Selector<()>)>,
```

Comment:
I would probably create a named struct that holds these items, instead of a 3-tuple?